src/mscorlib/src/System/Threading/Timer.cs

```diff
@@ -78,6 +78,7 @@ private static int TickCount
             [SecuritySafeCritical]
             get
             {
+#if !FEATURE_PAL
                 if (Environment.IsWindows8OrAbove)
```

Comment:
Why is IsWindows8OrAbove returning true?  I see a few other calls to it in mscorlib... should we ensure it returns false in these cases instead?  Or if not, do we need to address the other call sites?